tests/test_views_permissions.py

```diff
@@ -19,6 +19,44 @@ def get_app_settings(cls, extras=None):
         return settings
 
 
+class PermissionsUnauthenticatedViewTest(BaseWebTest, unittest.TestCase):
+
+    @classmethod
+    def get_app_settings(cls, extras=None):
+        settings = super().get_app_settings(extras)
+        settings['experimental_permissions_endpoint'] = 'True'
+        settings['bucket_create_principals'] = 'system.Everyone'
+        return settings
+
+
+class EntriesTestUnauthenticated(PermissionsUnauthenticatedViewTest):
+
+    def setUp(self):
+        super().setUp()
+        self.everyone_headers = get_user_headers('')
+        self.app.put_json('/buckets/beers', MINIMALIST_BUCKET,
+                          headers=self.everyone_headers)
+        self.app.put_json('/buckets/beers/collections/barley',
+                          MINIMALIST_COLLECTION,
+                          headers=self.everyone_headers)
+        self.app.put_json('/buckets/beers/groups/amateurs',
+                          MINIMALIST_GROUP,
+                          headers=self.everyone_headers)
+        self.app.put_json('/buckets/beers/collections/barley/records/{}'.format(RECORD_ID),
+                          MINIMALIST_RECORD,
+                          headers=self.everyone_headers)
+
+        # Other user.
+        self.app.put_json('/buckets/water', MINIMALIST_BUCKET,
+                          headers=get_user_headers('alice'))
+
+    def test_permissions_list_entries_for_everyone(self):
+        print(self.headers)
+        resp = self.app.get('/permissions', headers=self.everyone_headers)
+        permissions = resp.json['data']
+        self.assertEqual(len(permissions), 5)
```

Comment:
There needs to be a test somewhere that asserts the content of the bucket creation entry

Reply:
Will do